.python-version

```diff
@@ -0,0 +1 @@
+3.5.2
```

Comment:
@ccostino @anthonygarvan What do you think - can we lock this down? I think it'd provide some stability.